views/configure.pug

```diff
@@ -26,5 +26,46 @@ block content
       label(for='confirm-password').control-label Confirm password
       input(type='password',name='confirm-password',value='',required=true,data-match='#password',data-match-error="The password and the confirmation must match").form-control#confirm-password
       span.help-block.with-errors
+    div.consent_conf
+      +consent_form
+      div.form-group
+          div.checkbox
+            label.control-label
+              input(type='checkbox',name='agree_consent',id='register-agree-consent',value='1',required=true)
+              =_("I agree to grant consent to this research.")
     div.form-group
       button(type='submit').btn.btn-primary Configure
+    
+
+
+mixin consent_form
+  h2=_("Consent Form")
+
+  div.consent-form
+    p
+      | You are invited to participate in a research study on virtual assistants. With your consent, the Genie virtual assistant will record the commands you give it.
```

Comment:
For now, in almond-server it should talk about the Almond virtual assistant, not Genie

Reply:
fixed